packages/graphics/src/Graphics.ts

```diff
@@ -1024,7 +1024,7 @@ export class Graphics extends Container
             // but may be more than one plugins for graphics
             if (!DEFAULT_SHADERS[pluginName])
             {
-                const MAX_TEXTURES = renderer.plugins.batch.MAX_TEXTURES;
+                const MAX_TEXTURES = renderer.plugins[pluginName].MAX_TEXTURES;
```

Comment:
Small style improvement:
```suggestion
                const { MAX_TEXTURES } = renderer.plugins[pluginName];
```